strapi/strapi
# Feature/sizeofimage

### What does it do?

Describe the technical changes you did.
I changed the height and width of the svg to be ore visible 

### Why is it needed?

Describe the issue you are solving.
I anticipate that the SVG icon representing the file upload functionality within the application should be more prominent in size, which  would contribute to a more intuitive and user-friendly experience, as it ensures that users can readily discern and utilize this essential functionality without ambiguity or uncertainty.



### How to test it?

Provide information about the environment and the path to verify the behaviour.
Go to Strapi Dasboard --> Media Library
Click on "Add new assets"
See the image svg icon above "Drag & Drop here or" is tiny
Inspect the code for the svg


### Related issue(s)/PR(s)
Image SVG Icon too small when adding new assets to Media Library #16244

Let us know if this is related to any issue/pull request
